ui/src/tooltip.tsx

```diff
@@ -29,36 +30,37 @@ const
       flexGrow: 1
     },
     icon: {
-      color: '#323130',
+      color: cssVar('$text'),
       fontWeight: 400,
       fontSize: '14px',
       userSelect: 'none',
       textAlign: 'left',
       marginLeft: '0.5em',
       cursor: 'pointer'
+    },
+    preventOverflow: {
+      minWidth: 0,
+      flex: '1 1 0'
```

Comment:
I am wondering what is the role of `expand` now, since both CSS classes add `flex-grow` now.

Reply:
I kept it to avoid regression but it seems that we can have `flex-grow: 1` in every component. `min-width` is what prevents overflow. I'm on it.